src/taco-cli/cli/run.ts

```diff
@@ -235,6 +244,18 @@ class Run extends commands.TacoCommandBase {
         if (parsedOptions.options["debug"] && parsedOptions.options["release"]) {
             throw errorHelper.get(TacoErrorCodes.ErrorIncompatibleOptions, "--debug", "--release");
         }
+        
+        if (parsedOptions.options["livereload"] && parsedOptions.options["remote"]) {
```

Comment:
I rebased this branch on top of the master branch and tested again.
We are good to go with the merge into master.